.travis.yml

```diff
@@ -28,3 +31,10 @@ notifications:
     rooms:
       - secure: "YA1alef1ESHWGFNVwvmVGCkMe4cUy4j+UcNvMUESraceiAfVyRMAovlQBGs6\n9kBRm7DHYBUXYC2ABQoJbQRLDr/1B5JPf/M8+Qd7BKu8tcDC03U01SMHFLpO\naOs/HLXcDxtnnpL07tGVsm0zhMc5N8tq4/L3SHxK7Vi+TacwQzI="
 bundler_args: --path vendor/bundle
+
+# Railties specs on Ruby 1.8.7 are broken.
+# Last recorded passing CI run is https://travis-ci.org/rails/rails/builds/48782330
```

Comment:
Anyway, I'm not going to spend time to my fix railties tests on 1.8.7. I'm glad to remove this line, and we can merge this so that we fixes *most of the tests* but leave a few in red state.